infrastructure/stacks/dos_replica_etl/dos_replica_etl.tf

```diff
@@ -0,0 +1,145 @@
+resource "aws_lambda_function" "dos_replica_etl_lambda" {
+  filename         = data.archive_file.dos_replica_etl_function.output_path
+  function_name    = local.dos_replica_etl_function_name
+  description      = local.dos_replica_etl_description
+  role             = aws_iam_role.dos_replica_etl_lambda_role.arn
+  handler          = "dos_replica_etl.lambda_handler"
+  source_code_hash = data.archive_file.dos_replica_etl_function.output_base64sha256
+  runtime          = local.dos_replica_etl_runtime
+  timeout          = local.dos_replica_etl_timeout
+  memory_size      = local.dos_replica_etl_memory_size
+  publish          = true
+  tags             = local.standard_tags
+  layers           = [local.dos_replica_etl_core_dos_python_libs_arn]
+  environment {
+    variables = {
+      USR                = local.dos_replica_etl_db_user
+      SOURCE_DB          = local.dos_replica_etl_source_db
+      ENDPOINT           = local.dos_replica_etl_db_endpoint
+      PORT               = local.dos_replica_etl_db_port
+      REGION             = local.dos_replica_etl_db_region
+      SECRET_NAME        = local.dos_replica_etl_db_secret_name
+      SECRET_KEY         = local.dos_replica_etl_db_secret_key
+      ES_DOMAIN_ENDPOINT = local.es_domain_endpoint
+    }
+  }
+  vpc_config {
+    subnet_ids = [
+      data.terraform_remote_state.vpc.outputs.private_subnets[0],
+      data.terraform_remote_state.vpc.outputs.private_subnets[1],
+      data.terraform_remote_state.vpc.outputs.private_subnets[2]
+    ]
+    security_group_ids = [
+      local.dos_replica_etl_vpc_security_group,
+      aws_security_group.dos_replica_etl_security_group.id
+    ]
+  }
+}
+
+resource "aws_security_group" "dos_replica_etl_security_group" {
+  name        = "uec-sf-${var.profile}-dos-replica-etl_sg"
+  description = "Allows outbound traffic from ETL lambda"
+  vpc_id      = data.terraform_remote_state.vpc.outputs.vpc_id
+
+  egress = [
+    {
+      from_port        = 0
+      to_port          = 0
+      protocol         = "-1"
+      cidr_blocks      = ["0.0.0.0/0"]
+      ipv6_cidr_blocks = null
+      description      = null
+      prefix_list_ids  = null
+      security_groups  = null
+      self             = null
+    }
+  ]
+}
+
+resource "aws_security_group_rule" "dos_replica_etl_security_group_rule" {
+  type                     = "ingress"
+  from_port                = 443
+  to_port                  = 443
+  protocol                 = "tcp"
+  description              = "allows lambda etl process to insert data into ES"
```

Comment:
Update description:
"Allows SF Services ETL Lambda to access SF Elasticsearch datastore"